### DISCUSSION TITLE
Nested evaluations in Github Actions

### DISCUSSION THREAD

**Discussion by xanthehammer** [community/community#133109] [Actions]
### Select Topic Area

General

### Body

In my workflow I need to access one of my env variables dynamically. I'm using a matrix to run the job on different areas of my application and I have function_app set to the name of the current application area. I have an env variable called function_app_PATH for each one of my app areas. However, trying to use the value of that variable is proving tough. This is just outputting env.ASN_API_PATH instead of the actual value. 

`cd "${{ format('env.{0}_PATH', matrix.function_app) }}`

**Comment by ChristopherHX** [comment]
you don't need nested evaluation in this case
e.g.
```yaml
- run: |
    cd ${{ env[format('{0}_PATH', matrix.function_app)] }}
```
